README.md

```diff
@@ -500,11 +500,11 @@ Rather than reduce mere boilerplate code, look for variations on patterns and se
 
 Any programmer at any level can reduce perceived duplication. It's not difficult, and it's not an unassailable goal of design - except when it does not create countervailing problems. And that assessment is a matter of a judgment call that has to take into effect the unique conditions of the particular system being worked.
 
-If you reduce duplication of code patterns indiscriminately, you'll end up creating a _framework_ from which critical business logic cannot be extricated when the framework becomes too cumbersome to continue justifying its use. While not _all_ frameworks end up facing this fate, it's far more common than not, as framework developers struggle to maintain adoption and relevance as time goes passes by adding more features and specializations through abstraction rather than by extension (ie: plain old _vendor lock-in_).
+If you reduce duplication of code patterns indiscriminately, you'll end up creating a _framework_ from which critical business logic cannot be extricated when the framework becomes too cumbersome to continue justifying its use. While not _all_ frameworks end up facing this fate, it's far more common than not, as framework developers struggle to maintain adoption and relevance as time passes by adding more features and specializations through abstraction rather than by extension (ie: plain old _vendor lock-in_).
```

Comment:
The diff doesn't make it clear, but I removed the word "goes" in `relevance as time goes passes by adding more`.